pkgs/development/python-modules/langchain-openai/default.nix

```
@@ -28,14 +28,14 @@
 
 buildPythonPackage rec {
   pname = "langchain-openai";
-  version = "0.3.6";
+  version = "0.3.15";
```

```suggestion
  version = "0.2.5";
```
The previous change (to 0.3.6) was incorrect. See https://github.com/langchain-ai/langchain/releases

#350758 should have fixed this three days ago